recipes/spyder-screencast/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "spyder-screencast" %}
+{% set version = "0.0.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spyder-screencast-{{ version }}.tar.gz
+  sha256: d16993d9d5bd8842294859c6d0584360adf6b408ed57a716d57411ff009021af
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not win]
```

Comment:
Why isn't this recipe noarch: python?

Reply:
because of the pyqt==5.12.3, only the windows version has the Multimedia module incorporated, Linux and MacOS lack it ... 
so for the time being we can only do a windows package. Once the pyqt=5.15.x is there (we are going to work on that with Quansight and Anaconda) then we can enable all OS-es.